src/topojsonm/vendor/ezjsonm/ezjsonm.ml

```diff
@@ -0,0 +1,384 @@
+(*
```

Comment:
If we make a dependency on `Geojsone` then we can just use the module exported there rather that re-writing all of this code.

Reply:
Okay. How do we make a dependency on `Geojsone`? By including it in `dune` file?

Reply:
Yep, then the `topojsonm.opam` file